data-prepper-plugins/s3-source/src/test/java/org/opensearch/dataprepper/plugins/source/configuration/S3ScanBucketOptionTest.java

```diff
@@ -41,4 +41,11 @@ public void s3scan_bucket_options_with_scan_buckets_yaml_configuration_test() th
         assertThat(s3ScanBucketOption.getS3ScanFilter().getS3ScanExcludeSuffixOptions(),instanceOf(List.class));
         assertThat(s3ScanBucketOption.getS3ScanFilter().getS3ScanExcludeSuffixOptions().get(1),equalTo(".png"));
     }
+
+    @Test
+    public void s3scan_bucket_options_with_s3_prefix_test() throws JsonProcessingException {
+        final String bucketOptionsYaml = "              name: s3://test-s3-source-test-output";
```

Comment:
Do we need those spaces? If so, can we use something else (perhaps `\t`)?

Reply:
I wasn't able to use \t. I copied this config from existing test. Seeing the following error
```
while scanning for the next token
found character '\t(TAB)' that cannot start any token. (Do not use \t(TAB) for indentation)
```

Reply:
Do we need any spaces here?

Also, we have some other tests that work on in YAML using `---\n`. Maybe that?

But, we can keep this if it is the only thing that works.

Reply:
Okay, this is failing because of not having --- to indicate start of file. `---\n` worked